### DISCUSSION TITLE
text-embedding-3-large, error hugging face ???

### DISCUSSION THREAD

**Discussion by comeback01** [open-webui/open-webui#5786] [General]
Hello,

I made a mistake in the interface: I put "text-embedding-3-large" instead of "sentence-transformers/all-MiniLM-L6-v2". Since then, I can't start my Docker container because it tells me that it can't download "text-embedding-3-large" on Hugging Face. Do you have any idea how to solve the problem?

Thank you.

**Comment by thiswillbeyourgithub** [comment]
Take a look there https://docs.openwebui.com/getting-started/env-configuration/

Change probably RAG_EMBEDDING_MODEL or ENGINE

**Reply by comeback01** [reply]
When you don't know, it's always okay to say 'I don't know'... but in this case, I do know! The issue was in the SQLite database, and I had to update it manually to fix the model configuration.

**Comment by comeback01** [comment]
# Solution: How to Fix the "text-embedding-3-large" Error in a Docker Open WebUI Container

Hello,

I wanted to share the solution to a problem I recently encountered. I mistakenly set "text-embedding-3-large" instead of "sentence-transformers/all-MiniLM-L6-v2" in the interface, and since then, my Docker container wouldn’t start. It kept throwing an error saying it couldn’t download "text-embedding-3-large" from Hugging Face.

Here’s how I solved the issue by modifying the SQLite database directly in the container:

**Step 1: Access the Docker Container**

First, I accessed the SQLite database inside the Docker container. If your container isn’t running, you can start it with: `docker start <container_id>`. Then, to get into the container, I used `docker exec -it <container_id> /bin/bash`.

**Step 2: Extract and Modify the SQLite Database**

I copied the SQLite database from the container to my local environment to make the changes using the command: `docker cp <container_id>:/app/backend/data/webui.db /home/webui.db`. On my local machine, I used `sqlite3` to open the database and fix the wrong configuration.

1. Open the database with SQLite: `sqlite3 /home/webui.db`.
2. Update the model directly in the `config` table using this SQL command: `UPDATE config SET data = REPLACE(data, 'text-embedding-3-large', 'sentence-transformers/all-MiniLM-L6-v2') WHERE data LIKE '%text-embedding-3-large%';`.
3. Exit SQLite using `.exit`.

**Step 3: Copy the Database Back into the Container**

Once the modifications were done, I copied the updated database back into the container using the command: `docker cp /home/webui.db <container_id>:/app/backend/data/webui.db`.

**Step 4: Restart the Container**

Finally, I restarted the container to apply the changes with the command: `docker restart <container_id>`.

And that’s it! The container started successfully using the "sentence-transformers/all-MiniLM-L6-v2" model without any issues.

---

This should help you resolve the problem if you accidentally set the wrong model in Open WebUI.

---

I hope this helps others who face the same issue! 😊
